publishedbranches/docs-mongocli.yaml

```
@@ -27,7 +27,7 @@ version:
     - '1.2'
     - '1.1'
     - '1.0'
-  stable: '1.8'
+  stable: 'v1.8'
   upcoming: '1.9'
```

I'm actually about to do a version bump right now and this will be:
```suggestion
  stable: 'v1.9'
  upcoming: '1.10'
```